internal/docs/readme.go

```diff
@@ -17,96 +17,116 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
-
-// IsReadmeUpToDate function checks if the README file is up-to-date.
+// IsReadmeUpToDate function checks if all .md readme files are up-to-date.
 func IsReadmeUpToDate() (bool, error) {
-	logger.Debugf("Check if %s is up-to-date", ReadmeFile)
-
 	packageRoot, err := packages.MustFindPackageRoot()
 	if err != nil {
 		return false, errors.Wrap(err, "package root not found")
 	}
 
-	rendered, shouldBeRendered, err := generateReadme(packageRoot)
-	if err != nil {
-		return false, err
-	}
-	if !shouldBeRendered {
-		return true, nil // README file is static and doesn't use template.
-	}
-
-	existing, found, err := readReadme(packageRoot)
+	readmeFiles, err := ioutil.ReadDir(filepath.Join(packageRoot, "_dev", "build", "docs"))
 	if err != nil {
-		return false, errors.Wrap(err, "reading README file failed")
-	}
-	if !found {
-		return false, nil
-	}
-	return bytes.Equal(existing, rendered), nil
+		return false, errors.Wrapf(err, "failed to return a list of directory entries from %s", packageRoot)
+	}
+
+	for _, readme := range readmeFiles {
```

Comment:
In the same vein as https://github.com/elastic/elastic-package/pull/282/files#r591259737, how about splitting up this function into two?

Reply:
done :)